optaplanner-operator/src/k8s/openshift.yml

```diff
@@ -114,7 +114,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: solverreconciler-cluster-role
+  name: optaplannersolverreconciler-cluster-role
```

Comment:
The name is generated from the `*Reconciler` class; this file should have been updated after the class was renamed (which happened in some previous PR).